aries/aries_vcx_core/src/anoncreds/credx_anoncreds.rs

```diff
@@ -60,131 +59,47 @@ pub struct RevocationRegistryInfo {
     pub used_ids: HashSet<u32>,
 }
 
-/// Adapter used so that credx does not depend strictly on the vdrtools-wallet
-/// Will get removed when the wallet and anoncreds interfaces are de-coupled.
```

Comment:
I suppose this might be still relevant comment? The "adapter" thingie seems like an overhead.